crates/resolver/src/dns_sd.rs

```diff
@@ -47,6 +47,9 @@ impl<C: DnsHandle, P: ConnectionProvider<Conn = C>> DnsSdHandle for AsyncResolve
         let ptr_future = async move {
             let options = DnsRequestOptions {
                 expects_multiple_responses: true,
+                // TODO: This should use the AsyncResolver's options.edns0
```

Comment:
yes, it would be nice to thread this through. I this for this SD lookup, we don't need it, but it would be great for us to go back and clean up the EDNS handling.